src/modules/open-weather-api.js

```diff
@@ -0,0 +1,41 @@
+import { setData } from './weather-container';
+import { setCityPhoto } from './flickr-api';
+// "http://openweathermap.org/img/w/" + iconcode + ".png"; icon of json
+const searchWeatherByCity = (cityName) => {
+  setCityPhoto(cityName);
+  const url = 'https://api.openweathermap.org/data/2.5/weather?q='+
+              cityName+
+              '&appid='+process.env.WEATHER_API_KEY+'&units=metric';
+  getData(url);
+}
+const searchWeatherByCoordinates = (lat,long) => {
+  const url = 'https://api.openweathermap.org/data/2.5/weather?lat='+
+              lat+'&lon='+long+'&appid='+process.env.WEATHER_API_KEY+
+              '&units=metric';
+              console.log(lat,long);
+  getData(url);
+}
+
+const getData = (url) => {
+  fetch(url)
+  .then(response => response.json())
+  .then(json => {
+    console.log(json);
+      const obj = {
+        title: json.name,
```

Comment:
This should be in `setData` and if must be done here or any other place should use de structuring eg. `const { name } = json`